src/auth/component.ts

```diff
@@ -102,14 +97,18 @@ export default class ngeoAuthComponent extends LitElementI18n {
                 ? html`
                     <form name="logoutForm" role="form" @submit=${(evt: Event) => this.logout(evt)}>
                       <div class="form-group">
-                        <input type="submit" class="form-control btn prime" value=${i18next.t('Logout')} />
+                        <input
+                          type="submit"
+                          class="form-control btn prime"
+                          value=${i18next.t('Logout') as string}
+                        />
                       </div>
                       <div class="form-group">
                         <input
                           ?hidden="${!this.allowPasswordChange}"
                           type="button"
                           class="form-control btn btn-default"
-                          value=${i18next.t('Change password')}
+                          value=${i18next.t('Change password') as string}
```

Comment:
@ger-benjamin @sbrunner It removes errors in my IDE as the i18next object is not typed as a string as needed by the HTML spec, so casting it like that remove the error. It initially doesn't break the build or running the app, just getting dozens of errors that aren't relevant in my IDE makes it hard to spot real errors while working.

It is okay for you to do that for the translated string when needed ?

Reply:
Actually the CI does not like it ...

Reply:
It is an initial error from the lit-plugin, my fix made an typescript error...